stress-generator/values.yaml

```diff
@@ -0,0 +1,19 @@
+image:
+  repository: kupolua/stress-generator
+  tag: latest
+  env:
+    hostname: HOST
+    hostvalue: stress-demo-api.default.svc.cluster.local
```

Comment:
ok, but:
deployment: demo

"deployment field references of visualtext deployment, not stress test deployment